library/worldgen/biome/src/main/java/org/quiltmc/qsl/worldgen/biome/impl/modification/BiomeModificationImpl.java

```diff
@@ -56,6 +60,8 @@ public class BiomeModificationImpl {
 
 	private boolean modifiersUnsorted = true;
 
+	final BiomeModificationReloader reloader = new BiomeModificationReloader();
+
 	private BiomeModificationImpl() {
 	}
```

Comment:
I know it isn't your code, but:
```suggestion
	private BiomeModificationImpl() {}
```
why not :p